schema.sql

```diff
@@ -8,38 +8,40 @@ CREATE SCHEMA web3storage
     github TEXT,
     public_address TEXT NOT NULL,
     used_storage bigint DEFAULT 0,
-    inserted_at timestamp with time zone DEFAULT timezone('utc'::text, now()) NOT NULL
+    inserted_at timestamp with time zone DEFAULT timezone('utc'::text, now()) NOT NULL,
+    updated_at timestamp with time zone
   )
   CREATE TABLE auth_tokens (
     id BIGSERIAL PRIMARY KEY,
     name TEXT NOT NULL,
     secret TEXT NOT NULL,
     user_id bigint NOT NULL,
     inserted_at timestamp with time zone DEFAULT timezone('utc'::text, now()) NOT NULL,
```

Comment:
I'm assuming that `now()` is evaluated then a document is inserted and the default is applied and not when the table is created?